app/types/transport/requests/status/update-status.ts

```diff
@@ -0,0 +1,17 @@
+import { NewMessage } from "ctproto";
+
+export interface UpdateStatusPayload {
```

Comment:
why do we need to update statuses? I think, it is the separate issue (board-view sorting). 

The changing of task status from the popup should only change the `status` field of the `task`. Or not?

Reply:
same as
https://github.com/codex-team/task-manager/pull/104#discussion_r796102174